addons/stock_landed_costs/models/stock_move.py

```diff
@@ -0,0 +1,16 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class StockMove(models.Model):
+    _name = 'stock.move'
+    _inherit = 'stock.move'
+
+    def _get_price_unit(self):
+        self.ensure_one()
```

Comment:
This override may take precedence over the ones in `*_mrp`. In case of kit, we should share the invoice price unit over components. 
Also. The price unit is not read on purchase order line in some cases (return, standard_price, ... ) i guess we should do the same in this case
Maybe the best solution is to override `should_force_price_unit()` in case the move is linked to a landed cost invoice